generic/Emscripten-wasm.cmake

```diff
@@ -9,6 +9,10 @@
 #
 
 set(CMAKE_SYSTEM_NAME Emscripten)
+set(CMAKE_SYSTEM_VERSION 1)
+
+set(CMAKE_CROSSCOMPILING TRUE)
```

Comment:
Why? This is set automatically by CMake when `CMAKE_SYSTEM_NAME` is different from the host system name.

Reply:
I didn't actually know that 🤣